lib/msal-common/src/authority/Authority.ts

```diff
@@ -552,13 +552,15 @@ export class Authority {
                 : [];
             if (metadata.length === 0) {
                 // If no metadata is returned, authority is untrusted
+                this.logger.verbose("The network request completed successfully, but no metadata was returned.");
```

Comment:
Please log the response.body and response.status code here.

Reply:
I'd think this is warning verbosity.

Reply:
Any reason we can't just look at the network logs to determine what was returned? I'd rather not output response body here as it could be very large and make the logs hard to read.

Reply:
Bogdan, remember we talked about not logging an object?

Reply:
@Robbie-Microsoft - response.body is not an object, it's a string. It is in JSON format, so it can be deserialized into an object. Also, we should log it at least at warning level.
@tnorling - this only gets logged in error cases, where we're going to be throwing an authority is invalid exception. So it's likely that the app developer will open a bug / ICM and we need the logs :) .See the `if` condition above.

Reply:
@bgavrilMS I thought of a good solution. The reponse.body only has two properties: tenant_discovery_endpoint and metadata. If we reach this point, it'e because there is no metadata. Therefore, we only need to display the tenant_discovery_endpoint here.